src/gwt/src/org/rstudio/studio/client/workbench/Workbench.java

```diff
@@ -551,6 +558,38 @@ private void checkForLicenseMessage()
          globalDisplay_.showLicenseWarningBar(false, licenseMessage);
       }
    }
+
+   private void checkForCrashHandlerPermission()
+   {
+      boolean shouldPrompt = session_.getSessionInfo().getPromptForCrashHandlerPermission();
+      if (shouldPrompt)
+      {
+         String message =
+               "Would you like to enable automated crash reporting?\n\nIf RStudio crashes unexpectedly, the " +
```

Comment:
Suggest we make this a little simpler:

*May we upload crash reports to RStudio automatically? Crash reports don't include any personal information (except IP addresses for accounting) and can be turned off at any time.*

What do you think?